Gemfile

```diff
@@ -5,5 +5,5 @@ git_source(:github) { |repo_name| "https://github.com/#{repo_name}" }
 # Specify your gem's dependencies in danger-suggester.gemspec
 gemspec
 
+gem "danger-rubocop"
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/r7kamura/danger-suggester/blob/ea2fd5b752359dd3d35b7106ce4e106107afbbf3/Gemfile#L8"></span>Style/StringLiterals: Prefer single-quoted strings when you don’t need string interpolation or special symbols.</td>
    </tr>
  </tbody>
</table>